### DISCUSSION TITLE
How to use Re-Register Mode

### DISCUSSION THREAD

**Discussion by AntonK-Github** [okieselbach/Autopilot-Manager#16] [Q&A] [answered]
Hello Oliver, 

After Re-Register mode is enabled via 
AppConfig:AutopilotManagerConfig:UseReRegisterMode set to true

How then to use it? Does it need additional switch/parameter to be specified when calling ap.exe ?

Thank you.

**Comment by okieselbach** [comment, marked as answer]
I apologize, I totally oversaw the question. I hope you figured it out in the meanwhile. 
You don't have to do anything. It will automatically do all the magic in the background during the import request. If you have logging enabled on the AppService you can follow the log entries in "Log stream" and see the re-register working.
